app/models/user.rb

```diff
@@ -17,6 +17,11 @@ class User < ActiveRecord::Base
                           foreign_key: "followed_id"
   has_many :followers, through: :passive_follows
 
+  def feed
+    Tweet.where("user_id IN (?)", followed_users.ids)
+         .order('created_at DESC')
```

Comment:
Yes I could!

@BrandonMathis - hoping I didn't miss the point of using SQL here as you suggested. Can you take a look and let me know your thoughts?